opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -1839,7 +1839,7 @@ export const updateAttributeMetaResolved = async (context, user, initial, inputs
     return { element: initial };
   }
   // Check user access update
-  const manageAccessUpdate = updates.some((e) => e.key === 'authorized_members');
+  const manageAccessUpdate = updates.some((e) => e.key === 'authorized_members' || e.key === 'authorized_authorities');
```

Comment:
is it enough to add this test ? validateUserAccessOperation will only test if you have the right capa for changing authorized members and check current authorized members, but if there are no authorized members on the entity, it would grant you access. How would it keep from changing authorized_authorities when there are no authorized members ?

Reply:
it is done the line just below with validateUserAccessOperation

Reply:
to modify authorized_authorities --> have the capa SET_ACCESS ??
Is it the case only for organizations? or all the entities (ex : notif etc) ?